packages/lib/src/components/PageSizeSelector/index.tsx

```diff
@@ -40,7 +40,9 @@ const PageSizeSelector: React.FC<Props> = ({
   </StyledSelect>
 );
 
-const StyledSelect = styled(Select)`
+const StyledSelect = styled(
```

Comment:
Ohh I wonder if these errors might be due to lib is on a different `typescript` version than Spruce and Parsley (idk why that's the case)

Reply:
I tried updating the ts version and still see the same errors unfortunately :(